### PR TITLE
Qupzilla updated to 1.8.9

### PR DESCRIPTION
**Pull request by miqlas**

### CODE REVIEW

**Comment by waddlesplash** [comment]
bump.

File: www-client/qupzilla/qupzilla-1.8.9.recipe [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+PROVIDES="
+	qupzilla$secondaryArchSuffix = $portVersion
+	app:QupZilla$secondaryArchSuffix = $portVersion
+	"
+# TODO: update requirements to be more specific about which Qt
+# libraries are required.
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	qt5$secondaryArchSuffix >= 5.0.0
```
Comment (diversys) [inline www-client/qupzilla/qupzilla-1.8.9.recipe]: Isn't it just libqt5$secondaryArchSuffix now?

File: www-client/qupzilla/qupzilla-1.8.9.recipe [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+BUILD_PREREQUIRES="
+	cmd:qmake$secondaryArchSuffix
+	cmd:make
+	cmd:g++$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	qt5${secondaryArchSuffix}_devel >= 5.0.0
```
Comment (diversys) [inline www-client/qupzilla/qupzilla-1.8.9.recipe]: Same here.

**Comment by miqlas** [comment]
They changed the config paths, so i did a patch for that too. Let me know if it is not a good place to store the configs.

**Comment by DarkmatterVale** [comment]
bump

**Comment by miqlas** [comment]
Hovewer it saves the config files in correct folder and opens them at start (strace shows it), it doesn't work as it should: it doesn't keep your settings or doesn't reread them.

And it is a bit crashy :S

I don't really know, what is wrong, so don't merge it yet!

**Pull request closed without merging**